recipes/ray-packages/meta.yaml

```diff
@@ -0,0 +1,256 @@
+{% set version = "1.0.1" %}
+
+package:
+  name: ray-packages
+  version: {{ version }}
+
+source:
+  url: https://github.com/ray-project/ray/archive/ray-{{ version }}.tar.gz
+  sha256: e08ff04dc8bca99527dbc821446f8660cfe6cbc8c35db61410958b9aa9acee56
+  patches:
+    - patches/0001-Remove-setup-and-install-requirements-installed-by-c.patch
+    - patches/0002-Do-not-force-pickle5-in-sys.path.patch
+    - patches/0003-Fix-redis-build-for-non-default-compiler-drivers.patch
+    - patches/0004-Fix-plasma-linking-for-ancient-glibc.patch
+    - patches/0005-Redis-deps-now-build-but-do-not-link.patch
+    - patches/0006-Disable-making-non-core-entry-scripts.patch
+    - patches/0007-Ignore-warnings-for-one-file.patch
+    - patches/0008-Contain-bazel-root-and-output-dir-right-above-SRC_DI.patch
+    - patches/0009-Do-not-crash-if-BAZEL_SH-not-set-on-Windows.patch
+    - patches/0010-Convert-symlinks-to-junctions-on-Windows-before-buil.patch
+    - patches/0011-Add-workaround-for-os.path.isdir-on-Windows.patch
+    - patches/0012-Include-process.h-for-getpid-explicitly-on-Windows.patch
+
+build:
+  number: 0
+  skip: True  # [py<36]
+  # skip on MacOS as there's some weird compilation issue and I have no MacOS to develop on
+  skip: True  # [osx]
+
+# Need these up here for conda-smithy to handle them properly.
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cython >=0.29
+    - bazel <=3.4.1  # [not win]
+    - bazel  # [win]
+    - curl
+    - make
+    - m2-patch  # [win]
+
+outputs:
+  - name: ray-all
+    requirements:
+      host:
+      run:
+        - python
+        - {{ pin_subpackage('ray-core', exact=True) }}
+        #- {{ pin_subpackage('ray-debug', exact=True) }} # temp turned off, see below
+        - {{ pin_subpackage('ray-dashboard', exact=True) }}
+        # some rllib deps are missing on Windows
+        - {{ pin_subpackage('ray-rllib', exact=True) }}  # [not win]
+        - {{ pin_subpackage('ray-serve', exact=True) }}
+        - {{ pin_subpackage('ray-tune', exact=True) }}
+        # autoscaler does not work on Windows, no point of building it
+        - {{ pin_subpackage('ray-autoscaler', exact=True) }}  # [not win]
+    test:
+      imports:
+        # dummy test; actual tests are in subpackages
+        - ray
+
+  - name: ray-core
+    script: build-core.sh  # [not win]
+    script: build-core.bat  # [win]
```

Comment:
why not to use default script names and remove these lines?

Reply:
Mainly because this is a multi-output recipe, and you need to distinguish which script gets used for which output...